### DISCUSSION TITLE
Welcome to Feliz Discussions!

### DISCUSSION THREAD

**Discussion by Zaid-Ajaj** [Zaid-Ajaj/Feliz#302] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 64J0** [comment]
Nice man, this is a great feature to get in touch with the users, congrats.
